frontend/app/components/page-header-brand.tsx

```diff
@@ -20,7 +20,9 @@ export function PageHeaderBrand() {
         <section id="wb-lng">
           <h2 className="sr-only">{t('gcweb:header.language-selection')}</h2>
           <LanguageSwitcher>
-            <span className="hidden md:block">{t('gcweb:language-switcher.alt-lang')}</span>
+            <span className="hidden md:block" lang={t('gcweb:language-switcher.alt-lang-abbr')}>
```

Comment:
I feel we should define a new i18n key to represent the ISO language code rather than reusing `language-switcher.alt-lang-abbr`, which could theoretically change to something like "ENG"/"FRA" and break the use of the `lang` attribute.

Reply:
Sure, that makes sense. Implemented in commit 0acb6c945b4688ab7dcd716bf2f31d762801bee0.